src/views/GroupSettingsView.java

```diff
@@ -293,6 +312,12 @@ public void propertyChange(PropertyChangeEvent evt) {
             String eventInfo = "<html><b>" + event.getEventName() + "</b><br>Start: " + event.getStartTime().format(formatter) +
                     "<br>End: " + event.getEndTime().format(formatter) + "</html>";
             recommendedEventLabel.setText(eventInfo);
+        } else if ("exportCalendarSuccess".equals(evt.getPropertyName())) {
```

Comment:
![image](https://github.com/user-attachments/assets/7b81381b-5c3a-424e-9908-9111549ac7ce)


This popup still occurs despite not being in the group settings view. This may be an issue with the PropertyName of the event, although it will require more investigation on your end.

Reply:
I've fixed this in my latest commit.